tools/manifest/test262.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import print_function
+
+from typing import Dict, Optional, Text, Tuple, Any, Callable
+
+import re
+
+# Matches trailing whitespace and any following blank lines.
+_BLANK_LINES = r"([ \t]*[\r\n]{1,2})*"
+
+# Matches the YAML frontmatter block.
+_YAML_PATTERN = re.compile(r"/\*---(.*)---\*/" + _BLANK_LINES, re.DOTALL)
+
+_STRIP_CONTROL_CHARS = re.compile(r'[\x7f-\x9f]')
+
+
+class TestRecord:
```

Comment:
Why do we have a class that only has static methods? If we aren't making this into an actual class I'd prefer to just have top-level functions.

Reply:
Will fix

Reply:
Done